src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsAllocateExecution.java

```diff
@@ -136,6 +140,10 @@ protected ExternalWorkspace run() throws Exception {
             throw new AbortException(message);
         }
 
+        ExternalWorkspaceActionImpl externalWorkspaceAction = new ExternalWorkspaceActionImpl(exws, flowNode);
+        flowNode.addAction(externalWorkspaceAction);
+        exws.setWorkspaceUrl(flowNode.getUrl() + externalWorkspaceAction.getUrlName());
```

Comment:
I would rather keep a single root action for it, but this approach may work as well